src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
-body {
+$text-color: #616070;
+$title-color: #060b35;
+$color-button: #00acdc;
+
+.body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    box-sizing: border-box;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $title-color;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    box-sizing: border-box;
+    color: #616070;
```

Comment:
Consider using the `$text-color` variable instead of the hardcoded color `#616070` for consistency. This will make it easier to manage color changes across the stylesheet.